src/change-of-program/schema.js

```diff
@@ -0,0 +1,120 @@
+import definitions from '../common/definitions';
+import _ from 'lodash';
+
+module.exports = {
```

Comment:
If you're using ES6 imports here, could consider using ES6 export syntax, instead of commonjs.